New script: ReadQRCode (#31323)
demisto/content: Packs/CommonScripts/Scripts/ReadQRCode/README.md

> Extracts the text from a QR code.
> This script uses the open-source library OpenCV and is restricted to it's capabilities.
> The output of this script includes the output of the script "extractIndicators" run on the text extracted from the QR code.
> 
> ## Script Data

Reviewer: ```suggestion

This script uses the open-source library OpenCV to extract text from QR codes.
The output of this script includes the output of the script "extractIndicators" run on the text extracted from the QR code.

## Script Data
```
